orchestra/google_apps/service.py

```diff
@@ -129,3 +129,47 @@ def delete_folder(self, folder_id):
         except errors.HttpError as error:
             logger.exception('An error occured: %s', error)
             return None
+
+    def list_folder(self, folder_id):
```

Comment:
I've never seen an issue for "add tests to this thing we should have tested before" that actually got done :-) I'll try to add them now.